rules/Php81/Rector/ClassConst/FinalizePublicClassConstantRector.php

```diff
@@ -60,6 +60,13 @@ public function getNodeTypes(): array
      */
     public function refactor(Node $node): ?Node
     {
+        /** @var Node\Stmt\Class_ $parentClass */
+        $parentClass = $this->betterNodeFinder->findParentByTypes($node, [Node\Stmt\Class_::class]);
+
+        if ($parentClass->isFinal()) {
```

Comment:
when parentClass is not a Class_, it need to be skipped early:

```suggestion
        $parentClass = $this->betterNodeFinder->findParentByTypes($node, [Node\Stmt\Class_::class]);

        if (! $parentClass instanceof Node\Stmt\Class_) {
            return null;
        }

        if ($parentClass->isFinal()) {
```

Reply:
Maybe I should check on `null` instead? 🤔